scripts/releaseChangelog.js

```diff
@@ -186,8 +193,8 @@ async function main(argv) {
   });
 
   const changelog = `
-## TODO RELEASE NAME
-<!-- generated comparing ${lastRelease}..${release} -->
```

Comment:
Is this comment useful? We never keep it in the CHANGELOG.

Reply:
From my past release experience this also seemed true 🤷

Reply:
Potential value here is that it helps to spot the error if we compare to the wrong branch/release

Reply:
That's because I was unsure the script would pick the correct branch/release So I added this line to let a chance to spot the error